tailwind.config.js

```diff
@@ -0,0 +1,111 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    extend: {
+      backgroundImage: {
+        'hero-underline-1': 'url(/icons/small-line.svg)',
+        'hero-underline-2': 'url(/icons/large-line.svg)',
+      },
+      backgroundPosition: {
+        under: 'bottom 0px right 15px',
+      },
+      colors: {
+        primary: {
+          100: '#E5E5E5',
+          200: '#CBCBCB',
+          300: '#B1B1B1',
+          400: '#979797',
+          500: '#7D7D7D',
+          600: '#636363',
+          700: '#494949',
+          800: '#2E2E2E',
+          900: '#141414',
+        },
+        secondary: { 400: '#9A9A9A' },
+      },
+      fontSize: {
+        3.75: '0.9375rem',
+        4.75: '1.1875rem',
+        7.5: '1.875rem',
+        9.5: '2.375rem',
+        11.75: '2.9375rem',
+        14.75: '3.6875rem',
+        18.25: '4.5625rem',
+        23: '5.75rem',
+      },
+      maxWidth: {
+        '1/2': '50%',
+        '1.05/2': '55%',
+      },
+      screens: {
+        '1.1xl': '80.0625rem'
```

Comment:
are you actually registering a custom device size ?

Reply:
If you want to customize the existing screen size you can take a look here.

https://tailwindcss.com/docs/screens#max-width-breakpoints or you can also switch to fixed https://tailwindcss.com/docs/screens#fixed-range-breakpoints